lib/components/SelectionControl/radio/radio-button/radio-button.stories.tsx

```diff
@@ -0,0 +1,87 @@
+import React from "react";
+import RadioButton from "./index";
+
+const RadioButtonMeta = {
+  title: "Components/Selection control/Radio/RadioButton",
+  component: RadioButton,
+  parameters: { layout: "centered" },
+  tags: ["autodocs"],
+  argTypes: {
+    id: {
+      description: "Unique identifier for the radio button element.",
+    },
+    defaultChecked: {
+      description: "Sets the initial checked state of the radio button.",
+      control: { type: "boolean" },
```

Comment:
Maybe it's better to add options for some argTypes in order to give the ability to toggle between some values? E.g. 
```
        defaultChecked: {
            description: "Sets the initial checked state of the radio button.",
            options: ["true", "false"],
            control: { type: "boolean" },
        },
```